fabric-dimensions-v1/src/main/resources/fabric.mod.json

```diff
@@ -1,13 +1,25 @@
 {
   "schemaVersion": 1,
   "id": "fabric-dimensions-v1",
+  "name": "Fabric Dimensions API (v1)",
   "version": "${version}",
   "license": "Apache-2.0",
+  "icon": "assets/icon.png",
```

Comment:
That looks wrong. I think you should copy/paste the icon from another module into `resources/assets/fabric-dimensions-v1/icon.png` and then reference it as `assets/fabric-dimensions-v1/icon.png`.

Reply:
Resolved in https://github.com/FabricMC/fabric/pull/1428/commits/c1d2ff1c4f7e60e9dc4c7626dc482ff14c4e2dcf